test/integration/rules/aria-required-attr/required-attr.html

```diff
@@ -34,9 +34,10 @@
 <span role="radio" id="pass8" aria-checked="true">I am GREEN!</span>
 <input type="number" role="spinbutton" id="pass9" value="10" />
 <input type="number" role="spinbutton" id="pass10" />
+<div role="scrollbar" id="pass11">ok</div>
+<div role="slider" id="pass12">ok</div>
+<div role="spinbutton" id="pass13">ok</div>
```

Comment:
This is because of aria-valuenow, correct? I'd prefer to leave this as is. ARIA 1.2 has taken the default valuenow off scrollbar and slider because it makes very little sense, and wasn't well supported. I suspect the same is true for spinbutton, but we can test that if you want.